hedera-node/src/main/java/com/hedera/services/state/submerkle/CustomFee.java

```diff
@@ -271,8 +268,8 @@ public FractionalFeeSpec(
 			if (denominator == 0) {
 				throw new IllegalArgumentException("Division by zero is not allowed");
 			}
-			this.numerator = numerator;
-			this.denominator = denominator;
+			this.numerator = Math.abs(numerator);
+			this.denominator = Math.abs(denominator);
```

Comment:
should we not throw an exception when the values provided are negative?

Reply:
Good idea, done!